chain/sugar.go

```diff
@@ -46,6 +46,6 @@ func (mp *MessageProducer) CreateMinerActor(owner, worker address.Address, sealP
 		Worker:        worker,
 		Owner:         owner,
 		SealProofType: sealProofType,
-		Peer:          pid,
+		Peer:          abi_spec.PeerID(pid),
```

Comment:
Constructor params have been extended to have a multi-address field (`abi.Multiaddr`), should update this method to accept one as a params and pass it here.